Gruntfile.js

```diff
@@ -51,13 +51,14 @@ module.exports = function(grunt) {
     }
   };
 
-  var prefixMatch = "\\n *(function |var |static )?";
+  var regexLF = (grunt.util.linefeed === "\r\n" ? "(\\u000D\\n)" : "\\n"); //get the correct Lf for the operating system
+  var prefixMatch = "\\n *(function |var |static )?".replace("\\n", regexLF);
   var varNameMatch = "[^(:;]*(\\([^)]*\\))?"; // catch function args too
   var nestedBraceMatch = ": \\{[^{}]*\\}";
   var typeNameMatch = ": [^;]*";
-  var finalMatch = "((" + nestedBraceMatch + ")|(" + typeNameMatch + "))?\\n?;";
-  var jsdocInit = "\\n *\\/\\*\\* *\\n";
-  var jsdocMid = "( *\\*[^\\n]*\\n)+";
+  var finalMatch = "((" + nestedBraceMatch + ")|(" + typeNameMatch + "))?\\n?;".replace("\\n", regexLF);
+  var jsdocInit = "\\n *\\/\\*\\* *\\n".replace("\\n", regexLF);
+  var jsdocMid = "( *\\*[^\\n]*\\n)+".replace("\\n", regexLF);
```

Comment:
Instead of all these replacements, why not just interpolate regexLF where it's needed?